app/src/main/java/com/ataulm/whatsnext/api/LetterboxdApiFactory.kt

```diff
@@ -99,18 +104,12 @@ private class AddAuthorizationHeaderInterceptor(private val apiSecret: String, p
     }
 
     private fun Request.generateHashedSignature(): String {
-        val urlEncodedBody = (body as? FormBody)?.toUrlEncodedString() ?: ""
+        val urlEncodedBody = body?.toUrlEncodedString() ?: ""
```

Comment:
PATCH requests aren't FormBody. I don't actually know what they are - none of the subclasses of RequestBody that ASide was showing me 🤷